### PR TITLE
Welcome Google Assistant to Broid family

### PR DESCRIPTION
**Pull request by killix**
Solve https://github.com/broidHQ/integrations/issues/6

### CODE REVIEW

File: integrations/broid-google-assistant/src/core/adapter.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        const content: any = R.path(["object", "content"], data)
+          || R.path(["object", "name"], data);
+        const type: any = R.path(["object", "type"], data);
+
+        // <speak><say-as interpret-as="cardinal">12345</say-as></speak>
+        if (content.startsWith("<speak>") && content.endsWith("</speak>")) {
+          ssml = true;
+        }
+
+        // TODO
```
Comment (dustinblackman) [inline integrations/broid-google-assistant/src/core/adapter.ts]: Plan on doing this before the merge?

File: integrations/broid-google-assistant/src/core/webHookServer.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+  // Run configuration methods on the Express instance.
+  constructor(options?: IAdapterHTTPOptions, logLevel?: string) {
+    super();
+    this.actionsMap = new Map();
+    this.host = options && options.host || "127.0.0.1";
+    this.port = options && options.port || 8080;
+    this.logger = new Logger("webhook_server", logLevel || "info");
+    this.express = express();
+    // this.ee = new EventEmitter.EventEmitter();
```
Comment (dustinblackman) [inline integrations/broid-google-assistant/src/core/webHookServer.ts]: Clean?

File: integrations/broid-google-assistant/src/core/webHookServer.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  // Configure API endpoints.
+  private routes(): void {
+    const router = express.Router();
+    // placeholder route handler
+    router.post("/", (req, res) => {
+      this.assistant = new actionsSdk
+        .ActionsSdkAssistant({request: req, response: res});
+      this.assistant.handleRequest(this.actionsMap);
+      // res.send("");
```
Comment (dustinblackman) [inline integrations/broid-google-assistant/src/core/webHookServer.ts]: Still need this?

File: integrations/broid-google-assistant/src/core/adapter.ts (hunk 1 above)
Comment (killix) [inline integrations/broid-google-assistant/src/core/adapter.ts]: No yet, i need to figure out what is exactly do ;)

**Pull request merged**